x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/import/create_rules_stream_from_ndjson.ts

```diff
@@ -57,7 +57,12 @@ export const validateRules = (
       return obj;
     }
 
-    const result = RuleToImport.safeParse(obj);
+    const result = RuleToImport.safeParse({
+      ...obj,
+      // Ignore the rule source field for now. A proper handling of this field
+      // will be added as part of https://github.com/elastic/kibana/issues/180168
+      rule_source: undefined,
```

Comment:
Why do we need to pass the `undefined` explicitly here?

Reply:
This is so we don't forget to add a proper handling of this field later on. This field is optional and might be easily missed.